client/components/domains/search-filters/dropdown-filters.jsx

```diff
@@ -41,7 +38,6 @@ export class DropdownFilters extends Component {
 
 	state = {
 		showPopover: false,
-		showOverallValidationError: false,
```

Comment:
This only ever changed to `true` when `hasValidationErrors` returned `true`, which doesn't happen anymore. So this state property can be removed.